tests/e2e/helper_test.go

```diff
@@ -137,6 +154,22 @@ func setupDSCInstance() *dsc.DataScienceCluster {
 	return dscTest
 }
 
+func setupServicemeshSubscription(name string, ns string) *ofapi.Subscription {
```

Comment:
Can we generalize these functions? Moving forward we might have multiple operators as dependencies. Redefining the functions should not be required.

Currently we have 3 dependent operators - Servicemesh, serverless and authorino

Reply:
Bad naming :) It is actually not servicemesh specific.